src/MocosSim.jl

```diff
@@ -94,4 +94,19 @@ function simulate!(state::SimState, params::SimParams, callback)
   nothing
 end
 
+function outsidefeed!(state::SimState, params::SimParams,time_limit::TimePoint=typemax(TimePoint), strain::StrainKind=ChineseStrain)
+  N = length(state.individuals)
+  individuals = 1:N
+
+  for infection_time in 0.0:params.travels_frequency:time_limit
+    if params.infection_travels_function(state, params, infection_time)::Bool 
+      person_id = sample(state.rng, individuals)
+      event = Event(Val(OutsideInfectionEvent), infection_time, person_id, strain)
+      push!(state.queue, event)
+    end
+  end
+
+end
```

Comment:
why was it moved here from `simstate.jl`?

Reply:
current version of the function requires that both `simstate` and `simparams` be defined in order to work.